### ISSUE TITLE
wynnexcavation b is not working properly

### ISSUE THREAD

**Issue by Laaffy** [Wynncraft/Issues#1444]
so it says the quest is started i try to to talk to the main guy it shows no text it is black then i cannot go on further so i am screwed atm until it is fixed what do i do i have tryied everything

**Comment by TheRedAnt** [comment]
Ive had this problem too

**Comment by Jo10843** [comment]
Me too

**Comment by rhbvkleef** [comment]
Same problem

**Comment by Klfrr** [comment]
Same with me, I made an account for this problem

**Comment by Yoavronen** [comment]
Me too

**Comment by P3BBL35** [comment]
They changed how this quest works. You have to get the Yellow Crystal Shard (only 1) from Wynn Excavation A in order to be able to start the quest. After you give the Crystal Shard to Lykron, the quest should work properly.

**Comment by bastianxx04** [comment]
this is not the issue, the issue is that we get into the dungeon just fine, but we can't get out again after we find the purple crystal.

**Comment by P3BBL35** [comment]
What does it say in your quest book? Does it say "Solve the puzzle"? Or, does it say, "Find the Crystal"? If it says "Solve the puzzle", you'll have to walk up to the door behind the archaeologist, and the quest book should update. If it doesn't say so, check your quest book to see if it says, "Find the Crystal".

**Comment by bastianxx04** [comment]
it says solve the puzzle, and i have been behind the door, yet nothing happens..

**Comment by P3BBL35** [comment]
You have to walk up, like, physically be touching the door. If you need help, tell me what world you're on and I can come help you

**Comment by bastianxx04** [comment]
i did touch the door, but i found a solution that no mods have reccomended before, the trigger for the quest is at the place where you fall down(previous puzzle)

**Comment by P3BBL35** [comment]
well, the touching the door thing worked for me, so...

**Comment by xEvah** [comment]
I'm getting the same problem. I randomly lost the Yellow Crystal Shard I earned by completing the WynnExcavation Site A quest, which means I can't do the WynnExcavation Site B quest. I tried to kill the mob Pharaoh Nabak again, but I only got a Yellow Crystal, and no Yellow Crystal Shard. Lykron doesn't seem to accept this. When I hold it in my hand, and click on him, the quest doesn't start at all. Can anyone help me?
